pkg/server/webhook.go

```diff
@@ -325,19 +325,19 @@ func mergeVolumeMounts(volumeMounts []corev1.VolumeMount, containers []corev1.Co
 
 func updateAnnotations(target map[string]string, added map[string]string) (patch []patchOperation) {
 	for key, value := range added {
+		keyEscaped := strings.Replace(key, "/", "~1", -1)
```

Comment:
great catch! is this something you could add a unit test for, to assert this behavior is present?